src/gf-oemid

```diff
@@ -71,7 +71,7 @@ coreos_gf_run_mount "${tmp_dest}"
 # * grub config
 # * BLS config (for subsequent config regeneration)
 # First, the grub config.
-if coreos_gf exists "/boot/efi"; then
+if [$(coreos_gf exists "/boot/efi") == 'true']; then
```

Comment:
Need spaces around `[` I think.

Reply:
thanks. also needed to put the `$()` in quotes